Core1Platform/02_CoreCapabilities.md

```diff
@@ -0,0 +1,51 @@
+    - Maintain your business data and do the accounting.   
+    - Implement and monitor your commerce processes.   
+    - Get, transform, and transfer data from service providers, shops, and fulfillment providers.   
+    - Reporting on products, individual KPIs, and statistics.    
+    - and much more...
+
+### ERP system is in use
+
+If there is an existing ERP system in use, the solution provides complete software for linking existing and legacy IT systems with modern markets in the B2C (Business to Customer) and B2B (Business to Business) sectors.   
+The average age of running ERP solutions are about 12 years. This means in most cases that you are not able to quickly follow new evolutions in e-commerce with your legacy ERP system. Changing of processes and data models are then extremely time consuming and expensive.  
+The flexible data models of the *Actindo Core1 Platform* enable varying data to be mapped, transformed, and transferred to other modules, your legacy ERP system as well as other third-party systems. The direction of the data flow is thereby determined individually by the process orchestration.  
+The *Actindo Core1 Platform* supports the transformation to a direct-to-consumer business with all benefits that provides running an own branded online shop. You can use the *Actindo Core1 Platform* to modernize your e-commerce, to stay up-to-date, and to implement new evolutions.  
+
+## Omni-channel integration
+
+There are many ways to do digital commerce, including not only your own online store but also numerous marketplaces. However, managing products separately in each channel requires huge effort. With the *Omni-Channel* module, you can connect your web store and all relevant marketplaces as well as payment service providers directly to the platform and manage them centrally. You can integrate various store systems, such as Shopify or Magento 2. <!---Magento heißt jetzt Adobe Commerce??-->  
+
+## Data transformation and process orchestration
+
+The data model of the *DataHub* module with ETL system (Extract, Transform, Load) allows maximum flexibility. It enables complex and varying data to be mapped, transformed, and transferred. The transfer is done via external services to other modules as well as external systems such as legacy ERP systems, marketplaces, payment and fulfillment providers, etc. 
+You can customize your individual data fields and sources for every single process and "translate" them individually for their receivers.  
+
+The *Actindo Workflows* module is a tool that allows you to design, control, map, and monitor your business processes. Based on your special and individual business rules, your company's processes and data can be mapped and made available however and whenever you want. You can easily adapt and automate processes in a graphical way and almost without any code needed. 
```

Comment:
The *Actindo Workflows* module vs. The Actindo *DB ad reporting* (s. Zeile 46).
Es ist leider gemischt in unserer Doku, aber öfter *Actindo Modulname* module...